src/clientside/config.cpp

```diff
@@ -6,15 +6,19 @@ namespace config {
 
 ActuatorMessage build_command_message() {
   return ActuatorMessage{
-      .ov101 = digitalRead(pinout::MISSILE_SWITCH_1),
-      .ov102 = digitalRead(pinout::MISSILE_SWITCH_2),
-      .ov103 = digitalRead(pinout::MISSILE_SWITCH_6),
+      .NV102 = digitalRead(pinout::MISSILE_SWITCH_1),
+      .NV103 = digitalRead(pinout::MISSILE_SWITCH_2),
+      .NV104 = digitalRead(pinout::MISSILE_SWITCH_3),
+      .NV105 = digitalRead(pinout::MISSILE_SWITCH_4),
+      .OV102 = digitalRead(pinout::MISSILE_SWITCH_5),
+      .FV102 = digitalRead(pinout::MISSILE_SWITCH_6),
+      .OV101v = digitalRead(pinout::MISSILE_SWITCH_7),
+      .FV101v = digitalRead(pinout::MISSILE_SWITCH_8),
```

Comment:
We should map switches 1-2 to purge, 3-4 to vent, 5-6 to press and 7-8 to injector